asserts/pool.go

```diff
@@ -196,6 +251,14 @@ func (gRec *groupRec) markResolved(ref *Ref) (marked bool) {
 	return true
 }
 
+func (gRec *groupRec) markResolvedSeq(atseq *AtSequence, sequence int) (marked bool) {
```

Comment:
this is strangely never exercised by tests, also is missing the logic about errors from the other markResolved